docs/src/4.15.GLText.mdx

```diff
@@ -7,3 +7,29 @@ import { GLText } from "./jsx/allLiveEditors";
 The prop types are identical to `<Text />`, except that additionally each Text object supports an optional `billboard?: boolean` property. If true (the default), text will always face the camera; if false, text may be rotated in 3D according to `pose.orientation`.
 
 <GLText />
+
+## Scale Invariance
+
+`<GLText />` also supports rendering text at a constant scale regardless of the zoom level by using the optional `scaleInvariantFontSize?: number` property. If set (default is `undefined`), the text will be rendered always at the same size, resembling the behavior of the `<Text />` command. 
+
+Please note that `scaleInvariantFontSize` only works if the `billboard` property is set to true.
+
+<GLTextScaleInvariant />
+
+## Resolution
+
+By default, `<GLText />` uses a high resolution font texture atlas for rendering text, which may have an impact on the performance of your application. 
+
+The `resolution?: number` optional property allows to override this behavior by providing a custom character resolution value. If your application does not need to render text that is too close the camera, it is recommended to reduce the font resolution to ensure a better performance.
+
+Default value is `resolution={160}`. Minimum value is `resolution={40}`. 
+
+## Alphabet
+
+`<GLText />` generates a new texture atlas on the fly whenever a merker's text contains at least one character that was not rendered before. This is a complex operation and may impact on the performance of your application. 
+
+For that reason, is recommended to use the optional `alphabet?: string[]` property to provide a list of characters that are allowed as part of any marker's text. 
```

Comment:
```suggestion
In order to avoid texture generation when you have an expected subset of characters, you can use the optional `alphabet?: string[]`. 
```